src/UsedTxOuts.purs

```diff
@@ -0,0 +1,114 @@
+module UsedTxOuts
```

Comment:
Please add more comments into the module header, describing its purpose.

Reply:
ok